script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_3.txt

```diff
@@ -1,6 +1,6 @@
 <Page1>
 No, I'll just stay in my room.
-It would be a shame to lose this feeling of satisfaction by walking around since my body is now nicely relaxed after taking a bath.
+It would be a shame to lose this feeling of satisfaction by walking around since my body is now nicely relaxed after taking a bath. //Gramatically, I think there should be a comma after "around," but I think it breaks up the flow of the sentence a little too much.
```

Comment:
Since my body is now relaxed after taking a bath, it would be a shame to lose this feeling of satisfaction by walking around.